dali/sasha/saxref.cpp

```diff
@@ -188,15 +192,24 @@ struct cFileDesc // no virtuals
         return memcmp(key,name+1,sl)==0;
     }
 
-    StringBuffer &getName(StringBuffer &buf)
+    bool getName(StringBuffer &buf)
```

Comment:
trivial/not new: should be a const method really (as should get some of the members below)